cmake/Modules/SelectModule.cmake

```diff
@@ -28,20 +28,20 @@ macro (SelectModule)
   option(USE_TIFF "Use TIFF codec" ON)
   option(USE_JPEG "Use JPEG codec" ON)
   option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
-  option(DEBUG_MODE "Debug or release?" ON)
+  option(DEBUG_MODE "Debug or Release?" ON)
   option(BUILD_EXTENSIONS "Build extension library" ON)
   option(BUILD_EDITOR_SPINE "Build editor support for spine" ON)
   option(BUILD_EDITOR_COCOSTUDIO "Build editor support for cocostudio" ON)
   option(BUILD_EDITOR_COCOSBUILDER "Build editor support for cocosbuilder" ON)
   option(BUILD_BOX2D "Build box2d external without using it for physics library" OFF)
-  option(BUILD_LUA_LIBS "Build lua libraries" ${BUILD_LUA_LIBS_DEFAULT})
-  option(BUILD_JS_LIBS "Build js libraries" ${BUILD_JS_LIBS_DEFAULT})
-  # need review
-  option(USE_EXTERNAL_PREBUILT_LIBS "Use prebuilt libraries in external directory" ${USE_PREBUILT_LIBS_DEFAULT})
-  option(USE_COCOS_PREBUILT_LIBS "use cocos static libraries, return error if not find" ${USE_COCOS_PREBUILT_LIBS_DEFAULT})
-  option(USE_SOURCES_EXTERNAL "Use sources in external directory (automatically ON when USE_EXTERNAL_PREBUILT_LIBS is ON)" ${USE_SOURCES_EXTERNAL_DEFAULT})
+  option(USE_LUACOCOS2D "Build lua libraries" ${USE_LUACOCOS2D_DEFAULT})
+  option(USE_JSCOCOS2D "Build js libraries" ${USE_JSCOCOS2D_DEFAULT})
```

Comment:
I think new names is not good as before.

Reply:
agree, keep the variable name as before is a better choice